frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.unit.spec.js

```diff
@@ -15,7 +15,6 @@ describe("updateParametersWidgetStickiness", () => {
 
     const dashboard = {
       parametersWidgetRef: { offsetTop },
-      parametersAndCardsContainerRef: { style: {} },
```

Comment:
update tests since we don't use this anymore. This was used when we weren't using `position: sticky` and rely on JS positional calculations.